<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wiz_add_points</class>
 <widget class="QWizard" name="wiz_add_points">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Add Points</string>
  </property>
  <property name="options">
   <set>QWizard::HaveHelpButton</set>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Type of points</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Boundary Point</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Survery Point</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>107</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Control Point</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Source</string>
    </property>
    <widget class="QLineEdit" name="txt_file_path">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_browse_file">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbo_data_source_type">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>281</width>
       <height>33</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CSV File</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Excel File</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QGIS Layer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>External Layer</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delimiter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_delimiter">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cbo_longitude">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>121</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbo_latitude">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>121</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
